croud/util.py

```
@@ -76,3 +76,14 @@ def open_page_in_browser(url: str) -> int:
         except FileNotFoundError:  # WSL might be too old
             pass
     return webbrowser.open_new_tab(url)
+
+
+def nested_dict_value(data: dict, key: str):
+    cursor = data
+    for k in key.split("."):
+        if k in cursor:
+            cursor = cursor[k]
+        else:
+            return None
```

I feel like this should raise a `KeyError` or something instead of only returning `None`.